src/goto-programs/remove_exceptions.cpp

```diff
@@ -197,7 +199,7 @@ void remove_exceptionst::instrument_exception_handler(
       symbol_table.lookup(id2string(function_id)+EXC_SUFFIX);
     // next we reset the exceptional return to NULL
     symbol_exprt lhs_expr_null=function_symbol.symbol_expr();
-    null_pointer_exprt rhs_expr_null((pointer_typet(empty_typet())));
+    null_pointer_exprt rhs_expr_null(pointer_type(empty_typet()));
```

Comment:
Looks like a most vexing parse, but maybe the final set of brackets on `empty_typet()` stop it from being interpreted as a function declaration.

In any case these might be clearer using uniform-initialization, but that would be beyond the scope of this PR.